yarn.lock

```diff
@@ -0,0 +1,11003 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'd prefer not to introduce more tooling. The newest versions of npm can achieve the same things as yarn.